### PR TITLE
Link homepage with the assignments page

### LINKED ISSUE #45: link homepage with the assignments page
**Is your feature request related to a problem? Please describe.**
- We need to link the assignment page to the home page

**Describe the solution you'd like**
- link homepage with the assignments page 

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #45 

## Goals

- Link homepage with the assignments page

## Approach

- Link homepage with the assignments page

### Screenshots

Screenshot from 2020-04-05 11-56-41
  
### Preview Link
https://anjula-sack.github.io/sack-site/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -249,8 +249,23 @@
...
+	<section class="section">
+		<div class="container">
+			<div class="row">
+				<div class="col-md-4 col-sm-4 no-padd">
+					<div class="feature-box4 padd-30 bg-blue no-margin">
+						<i class="icon icon-book-open"></i>
+						<h4>Assignments</h4>
+						<p>To get the assignments for grade 6 to 13 please visit the assignment page using the button
+							below.</p>
+						<a href="apps/assignments/" class="btn radial btn-dark-blue hvr-icon-wobble-horizontal">Visit</a>
```
Comment (jayasanka-sack) [inline index.html:263]: Remove this extra line

Comment (jayasanka-sack) [reply index.html:263]: Let's hard wrap the code. :)

Also, shall we remove this? ("St. Anthony's College" text) I think it doesn't make sense. Let's separate with this using a `<hr/>` tag.
Screenshot 2020-04-05 at 12 10 50

**Comment by anjula-sack** [comment]
> Also, I think it's for the whole school. Let's add a generic text like "To get the assignments, please visit the assignments page using the button below."

okay :)

File: index.html [hunk 2]
```diff
@@ -249,19 +249,25 @@
...
+	<br/>
+	<section class="section">
+		<div class="container">
+			<div class="row">
+				<div class="col-md-4 col-sm-4 no-padd">
+					<div class="feature-box4 padd-30 bg-blue no-margin">
+						<i class="icon icon-book-open"></i>
+						<h4>Assignments</h4>
+						<p>To get the assignments, please visit the assignments page using the button below.</p>
+						<a href="apps/assignments/" class="btn radial btn-dark-blue hvr-icon-wobble-horizontal">Visit</a>
```
Comment (jayasanka-sack) [inline index.html:261]: Let's hard wrap the code at 100th character

**Comment by jayasanka-sack** [comment]
@all-contributors please add @anjula-sack  for code

**Pull request merged**